boards/arm/stm32l1_disco/stm32l1_disco_defconfig

```diff
@@ -4,7 +4,6 @@ CONFIG_ARM=y
 CONFIG_BOARD_STM32L1_DISCO=y
 CONFIG_SOC_SERIES_STM32L1X=y
 CONFIG_SOC_STM32L151XB=y
-CONFIG_CORTEX_M_SYSTICK=y
```

Comment:
Please move this fix to the board commit.

Reply:
sure, badly squished sorry.